### ISSUE TITLE
Mage Blink Bug: Teleports player outside of cave and through terrain.

### ISSUE THREAD

**Issue by akhial** [dalaranwow/dalaran-wow#5227]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Inside a cave in Thousand Needles (coords: 43,34) using blink teleports player outside of cave.

**Expected behaviour**: Tell us what should happen instead

Should be blocked by terrain keeping player in the cave.

**Steps to reproduce the problem**:

1. Stand inside the Thousand Needles cave at aproximate given coords and face outside the cave (towards Thousand Needles exterior).
2. Use Mage Blink.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Initial position inside the cave:
![1](https://user-images.githubusercontent.com/20596622/34496810-507b8660-effb-11e7-9437-0a13f457c820.jpg)

Result 1:
![2](https://user-images.githubusercontent.com/20596622/34496814-58936e30-effb-11e7-85b9-cd36c25ac471.jpg)

Result 2: Not having Mage Slow Fall or hearth would mean death.
![3](https://user-images.githubusercontent.com/20596622/34496819-5e7a3c66-effb-11e7-8a80-d061d93922f8.jpg)

**Comment by Rushor** [comment]
duplicate and fix for the issue here: https://github.com/dalaranwow/dalaran-wow/issues/4496
